components/clp-package-utils/clp_package_utils/controller.py

```diff
@@ -750,6 +782,8 @@ def _set_up_env(self) -> None:
         env_vars |= self._set_up_env_for_query_worker(num_workers)
         env_vars |= self._set_up_env_for_reducer(num_workers)
         env_vars |= self._set_up_env_for_webui(container_clp_config)
+        if self._clp_config.mcp_server is not None:
+            env_vars |= self._set_up_env_for_mcp_server()
```

Comment:
see above suggestion for skipping the component creation
```suggestion
        env_vars |= self._set_up_env_for_mcp_server()
```